# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderBufferTextureHostSWGL.cpp
# Commit: 9d637c05af7f
# Full Hash: 9d637c05af7f8c39af78b290335ca55994849ee8
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r ea88b243717a -r 9d637c05af7f gfx/webrender_bindings/RenderBufferTextureHostSWGL.cpp
--- a/gfx/webrender_bindings/RenderBufferTextureHostSWGL.cpp	Fri Nov 06 00:00:47 2020 +0000
+++ b/gfx/webrender_bindings/RenderBufferTextureHostSWGL.cpp	Fri Nov 06 00:01:36 2020 +0000
@@ -69,7 +69,8 @@
   }
 }
 
-bool RenderBufferTextureHostSWGL::MapPlane(uint8_t aChannelIndex,
+bool RenderBufferTextureHostSWGL::MapPlane(RenderCompositor* aCompositor,
+                                           uint8_t aChannelIndex,
                                            PlaneInfo& aPlaneInfo) {
   switch (mDescriptor.type()) {
     case layers::BufferDescriptor::TYCbCrDescriptor: {